sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/testing/BigqueryClient.java

```diff
@@ -114,6 +114,8 @@
   private static final Logger LOG = LoggerFactory.getLogger(BigqueryClient.class);
   // The maximum number of retries to execute a BigQuery RPC
   static final int MAX_QUERY_RETRIES = 4;
+  // How long should BQ jobs.query call wait for query completion. Default is 10 seconds.
```

Comment:
Did you mean to say default is 20 seconds?

Reply:
No, the default (if unset) is 10 seconds.
https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/query#QueryRequest